ouroboros-network/src/Ouroboros/Network/Block.hs

```diff
@@ -128,7 +128,7 @@ class ( Eq        (HeaderHash b)
       , NoUnexpectedThunks (HeaderHash b)
       ) => StandardHash b
 
-data ChainHash b = GenesisHash | BlockHash (HeaderHash b)
+data ChainHash b = GenesisHash | BlockHash !(HeaderHash b)
```

Comment:
Hah, I'm surprised to discover there was no bang here. I guess we only recent started storing a `ChainHash` in some state (or we used `$!` in some places).